### ISSUE TITLE
Unsatisfied behavior of pre-train model

### ISSUE THREAD

**Issue by ChuanjunLAN** [thunil/TecoGAN#5]
Hi, 
Thanks for your great job and share. To my opinion, your creation about PP Loss, Spatio-temporal discriminator would help the behavior of SR task intuitively. 
I've tried run the inference process with the pre-trained model you guys supported, for the set calendar, it has not bad behavior, but for other set, like the video you showed on youtube, the result not satisfies me. I noticed that the compare is between LR and TecoGAN's output, and i tried to use method VESPCN, FRVSR... For the perceptual side, I've got Teco's output is no better than FRVSR(960x540-4k); For the artifacts side, I've got that the Teco's output is not better than VESPCN(when i saw your paper,i was excited,cause i think you might solved partly the problem of instable of GANS).
So,will you guys give me some advice about my confusion? Is unsatisfied behaviors cause of the influence of pre-train model? If you guys can give me some advice about the training step, i answer my question by myself.

Thanks in advance,
chuanjun

**Comment by WSure00** [comment]
hello , i have some problem in running this project.where do you run this program, on Win or Linux?
when i try to run the program ,i have trouble in connecting to the url，i can't download the model and resources.( i am located in china)

**Comment by ChuanjunLAN** [comment]
> hello , i have some problem in running this project.where do you run this program, on Win or Linux?
> when i try to run the program ,i have trouble in connecting to the url，i can't download the model and resources.( i am located in china)

Hi, I tested the repo both on windows and Linux, i did not meet the problem you mention above.
I think it might be "the firewall" problem. Maybe you can ask some friends to help you download all the resources and retry~

**Comment by WSure00** [comment]
OK，thanks




    


On 04/22/2019 15:03, ChuanjunLAN wrote: 
hello , i have some problem in running this project.where do you run this program, on Win or Linux?
when i try to run the program ,i have trouble in connecting to the url，i can't download the model and resources.( i am located in china)

Hi, I tested the repo both on windows and Linux, i did not meet the problem you mention above.
I think it might be "the firewall" problem. Maybe you can ask some friends to help you download all the resources and retry~

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/thunil/TecoGAN","title":"thunil/TecoGAN","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/thunil/TecoGAN"}},"updates":{"snippets":[{"icon":"PERSON","message":"@ChuanjunLAN in #5: \u003e hello , i have some problem in running this project.where do you run this program, on Win or Linux?\r\n\u003e when i try to run the program ,i have trouble in connecting to the url，i can't download the model and resources.( i am located in china)\r\n\r\nHi, I tested the repo both on windows and Linux, i did not meet the problem you mention above.\r\nI think it might be \"the firewall\" problem. Maybe you can ask some friends to help you download all the resources and retry~"}],"action":{"name":"View Issue","url":"https://github.com/thunil/TecoGAN/issues/5#issuecomment-485345594"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/thunil/TecoGAN/issues/5#issuecomment-485345594",
"url": "https://github.com/thunil/TecoGAN/issues/5#issuecomment-485345594",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by ch135** [comment]
Excuse me, can i ask you some question. Fistly,  how did you convert the video into an image for super resolution? And, is it a composite video of super-resolution images?

I have been learning image super resolution before, knowing nothing about video super resolution, I hope to receive your reply in your busy schedule. Thank you very much.

**Comment by whyboris** [comment]
@ch135 -- look into `ffmpeg` for example [this article](https://www.bugcodemaster.com/article/extract-images-frame-frame-video-file-using-ffmpeg)

Once you extract the images and upscale them with this (or any other method) you can stitch them back together with `ffmpeg` again 👍

**Comment by WSure00** [comment]
OK，thank you！




    


On 06/15/2019 05:52, Boris wrote: @ch135 -- look into ffmpeg for example this article
Once you extract the images and upscale them with this (or any other method) you can stitch them back together with ffmpeg again 👍

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/thunil/TecoGAN/issues/5?email_source=notifications\u0026email_token=AJN3VUNL3JZ5ID4M5HDBHHDP2QHLBA5CNFSM4HHIXDIKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXYF2XY#issuecomment-502291807",
"url": "https://github.com/thunil/TecoGAN/issues/5?email_source=notifications\u0026email_token=AJN3VUNL3JZ5ID4M5HDBHHDP2QHLBA5CNFSM4HHIXDIKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXYF2XY#issuecomment-502291807",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
